tools/site_admin/sitenews.php

```diff
@@ -375,7 +373,7 @@ function show_all_news_items_for_page($news_page_id)
             echo "<tr>";
             echo "<td class='commands'>";
             echo "<p><b>" . _("Posted") . ":</b><br>";
-            echo strftime(_("%B %e, %Y"), $news_item['date_posted']) . "</b>";
+            echo icu_date_template("medium", $news_item['date_posted']) . "</b>";
```

Comment:
Again, no objection to the change; just making sure that adding the abbreviation of the day of week was intentional.

Reply:
It was sort of intentional. Looking at it now I'm inclined to change it to the `short` form. That will abbreviate the day of the month but remove the day of the week. This is also the only instance of the `medium` form which means we could remove it. Thoughts?

Reply:
Done -- no more `medium` form.